modules/UI/recording/Recording.js

```diff
@@ -492,21 +497,32 @@ const Recording = {
                     this.eventEmitter.emit(
                         UIEvents.RECORDING_TOGGLED,
                         { streamId });
-                    sendAnalyticsEvent(RECORDING_STARTED);
+
+                    // The confirm button on the start recording dialog was
+                    // clicked
+                    sendAnalytics(
+                        createRecordingDialogEvent(
+                            'start',
+                            'confirm.button'));
                 })
                 .catch(reason => {
                     if (reason === APP.UI.messageHandler.CANCEL) {
-                        sendAnalyticsEvent(RECORDING_CANCELED);
+                        // The cancel button on the start recording dialog was
+                        // clicked
+                        sendAnalytics(
+                            createRecordingDialogEvent(
+                                'start',
+                                'cancel.button'));
                     } else {
                         logger.error(reason);
                     }
                 });
             } else {
+                // Note that we only fire analytics events for Jibri.
```

Comment:
Why is that only analytics should be recorded for jibri? Why not both?

Reply:
We only support jibri, the other code is essentially dead. I was too lazy to add a recordingType to all events, and check the conditions for each event (i.e. is it actually "recording started" or "recording button clicked"), so I chose to remove them.